projects/demo/src/modules/components/slider/examples/2/index.html

```diff
@@ -1,8 +1,3 @@
-<tui-slider
-    [formControl]="testValue"
-    [keySteps]="keySteps"
-    [min]="5000"
-    [max]="1000000"
-    [segments]="3"
-    [steps]="3"
-></tui-slider>
+<input tuiSlider value="65" class="first" />
```

Comment:
We need examples with ngModel and with formControl too.

Reply:
fixed

Now we have examples with ngModel, with FormControl and with native attributes